.circleci/main.yml

```diff
@@ -1513,6 +1514,74 @@ jobs:
           command: |
             poetry version << parameters.version >>
             poetry publish --build --username "$PYPI_USERNAME" --password "$PYPI_PASSWORD_PACHYDERM_SDK"
+  label-studio-ci-testing-docker-build:
+    executor: python
+    steps:
+      - setup_remote_docker:
+          version: 20.10.18
+          docker_layer_caching: true
+      - run:
+          name: docker login
+          command: echo "$DOCKER_PWD" | docker login --username pachydermbuildbot --password-stdin
+      - run:
+          name: Clone Label Studio repo
+          command: git clone https://github.com/pachyderm/label-studio.git label-studio
+      - run:
+          name: Build Docker image
+          command: |
+            cd label-studio
+            DOCKER_BUILDKIT=1 docker build -t pachyderm/label-studio-ci-testing:$CIRCLE_SHA1 .
+      - run:
+          name: Docker push
+          command: docker push pachyderm/label-studio-ci-testing:$CIRCLE_SHA1
+  test-label-studio:
+    resource_class: xlarge
+    machine:
+      image: << pipeline.parameters.machine_image >>
+    environment:
+      LABEL_STUDIO_USERNAME: user@domain.com
+      LABEL_STUDIO_PASSWORD: pass
+      LABEL_STUDIO_USER_TOKEN: abcdef
+    working_directory: ~/project/label-studio
+    steps:
+      - checkout:
+          path: ~/project
+      - install-minikube
+      - install-helm
+      - install-kubectl
+      - start-minikube
+      - wait-for-minikube
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Install pachctl
+          command: |
+            sudo mv ./dist-pach/pachctl/pachctl_linux_amd64_v1/pachctl /usr/local/bin/pachctl
+      - run:
+          name: Install pach
+          command: helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=NodePort
```

Comment:
We probably need a wait-for-pachyderm-docker-build step here.(There should be one you can re-use) This will probably work pretty often because we are waiting for minikube, but I think there are situations where minikube comes up before the docker image is built and the helm install fails.